### PR TITLE
Update TUTORIAL.md to bring in line with the Google Cloud Blog

### PR DESCRIPTION
**Pull request by mtronci**
This pull requests contains a small change, aiming to bring the `TUTORIAL.MD` file in line with the Regulatory Reporting page on the [Google Cloud Architecture Centre](https://cloud.google.com/architecture/set-up-regulatory-reporting-architecture-bigquery#upload_the_sample_data).

### CODE REVIEW

**Comment by mescanne** [comment]
Looks good to me.

File: docs/TUTORIAL.md [hunk 1]
```diff
@@ -268,115 +284,95 @@ and how the `reporting_day` variable that you passed is used in the `control.rep
...
+     data load step, and push the container to Google Container Repository:
 
-2. Containerize the DBT code for the data transformation step, and push the container to Google Container Repository. 
-    ```
-    gcloud builds submit --config use_cases/examples/home_loan_delinquency/dbt/cloudbuild.yaml
-    ```
-    Containerization helps you to create a package that can be easily versioned and deployed. 
+     ```
+     cd ../data_load/     # Note the Dockerfile in this folder
+     gcloud builds submit --tag $GCR_DATALOAD_IMAGE  # Pushes the image to GCR
```
Comment (mescanne) [inline docs/TUTORIAL.md:344]: This is now gcloud builds submit --tag $HOMELOAN_GCR_DATALOAD

**Review by mescanne — CHANGES_REQUESTED** [review]

File: docs/TUTORIAL.md [hunk 2]
```diff
@@ -1,219 +1,230 @@
...
 
+12. To verify that an ingest bucket has been created, in the Google Cloud console,
+     go to the **GCS** page and check for a bucket with a name
+     that's similar to the value of `PROJECT ID`.
+    
 
-13. In the Google Cloud Console, go to the **Cloud Storage** page and check for a bucket with a name like 
-`${project}-${region}-ingest-bucket` to verify that an ingest bucket has been created.
-
+13. Go to the **Big Query** page and verify that the following
```
[the remark this answers is not in the document]
Comment (mtronci) [reply docs/TUTORIAL.md]: The instructions on the blog are still referring to home loan only, so we can leave it at this for now.

File: docs/TUTORIAL.md [hunk 3]
```diff
@@ -222,41 +233,46 @@ both the `homeloan_data` and `homeloan_expectedresults` datasets.
...
 
 
-2. Test the connection between your local dbt installation and
-   your BigQuery datasets by running the following command:
+2.  Test the connection between your local dbt installation and your
+    Big Query datasets:
+
     ```
-    dbt debug 
+    dbt debug --profiles-dir profiles/
```
[the remark this answers is not in the document]
Comment (mtronci) [reply docs/TUTORIAL.md]: yes, now fixed!

File: docs/TUTORIAL.md (hunk 1 above)
Comment (mtronci) [reply docs/TUTORIAL.md]: this should be `gcloud builds submit --config use_cases/examples/home_loan_delinquency/data_load/cloudbuild.yaml`

**Comment by mtronci** [comment]
thanks Mark, I have re-synced the tutorial aligning with the latest [version on the Cloud Architecture Centre](https://cloud.google.com/architecture/set-up-regulatory-reporting-architecture-bigquery).
That should resolve all (but one) comments

**Review by mescanne — APPROVED** [review]

**Pull request merged**